src/components/comments/CommentSection.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Question from "./Question.js";
+import Answer from "./Answer.js";
+export default function CommentSection(props) {
+  let followupSection = [];
+  for (let i = 0; i < props.followups.length; i++) {
```

Comment:
Not a huge deal, but I think you can write this code a bit more concisely with [Array.map()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) rather than the `for` syntax, e.g. 

```
props.followups.length.map((followup) => {
  ...
})
```

Reply:
Cool, that cleans the code up a lot!